stock_picking_batch_extended/static/description/index.html

```diff
@@ -380,7 +380,11 @@ <h1 class="title">Stock batch picking</h1>
 <li>Check availability and reserve quants for all pickings</li>
 <li>Mark all pickings as done when delivery is done.</li>
 <li>Make partial delivery by filling done quantities in pack operations tab.</li>
+<li>Generate invoices for sales orders related to the pickings when the batch
```

Comment:
don't touch index and Readme.rst files you need to modify only files in readme folder

Reply:
In this case, he's running locally oca-gen-readme, so it's OK to have these changes all together.